nucliadb_protos/utils.proto

```diff
@@ -109,4 +109,9 @@ message UserVectorsList {
 enum VectorSimilarity {
     COSINE = 0;
     DOT = 1;
-}
\ No newline at end of file
+}
+
+enum ReleaseChannel {
```

Comment:
Wdyt about making it explicit that this is for the shard/node?
`NodeReleaseChannel`? or `ShardReleaseChannel`?